src/resources/views/crud/list.blade.php

```diff
@@ -30,15 +30,15 @@
         <div class="row mb-0">
           <div class="col-sm-6">
             @if ( $crud->buttons()->where('stack', 'top')->count() ||  $crud->exportButtons())
-              <div class="hidden-print {{ $crud->hasAccess('create')?'with-border':'' }}">
+              <div class="d-print-none {{ $crud->hasAccess('create')?'with-border':'' }}">
 
                 @include('crud::inc.button_stack', ['stack' => 'top'])
 
               </div>
             @endif
           </div>
           <div class="col-sm-6">
-            <div id="datatable_search_stack" class="mt-sm-0 mt-2"></div>
+            <div id="datatable_search_stack" class="mt-sm-0 mt-2 d-print-none"></div>
```

Comment:
Just `print` fixes - looks like we didn't change the classes when moving from Bootstrap 3 to Bootstrap 4.